Composer/packages/client/src/components/Notifications/SurveyNotification.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
This file should be called `useSurveryNotification.ts` since it is a hook and doesn't have tsx components.

Reply:
Good idea. This was originally a component until I realized it didn't render anything and could just turn into a hook.